content-src/components/Search/Search.jsx

```diff
@@ -119,6 +119,7 @@ export class _Search extends React.PureComponent {
       {!this.props.handoffEnabled &&
       <div className="search-inner-wrapper">
         <label htmlFor="newtab-search-text" className="search-label">
+          {/* Remove this sr-only span? */}
```

Comment:
Yeah, I believe this whole `label` can be removed. devtools accessibility seems to be happy enough with the actual `input`'s `title`